charts/node-analyzer/templates/_helpers.tpl

```diff
@@ -0,0 +1,209 @@
+{{- printf "requests:\n  cpu: %s\n  memory: %s\nlimits:\n  cpu: %s\n  memory: %s" $largeCpu $largeMemory $largeCpu $largeMemory -}}
+{{- else -}}{{/* "custom" or anything else falls here */}}
+{{- toYaml .Values.resources -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper Sysdig Agent image name for module building
+*/}}
+{{- define "nodeAnalyzer.image.kmodule" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.slim.kmoduleImage.repository -}} {{- if .Values.slim.kmoduleImage.digest -}} @ {{- .Values.slim.kmoduleImage.digest -}} {{- else -}} : {{- .Values.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper nodeAnalyzer Agent image name for the Runtime Scanner
+*/}}
+{{- define "nodeAnalyzer.image.runtimeScanner" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.runtimeScanner.image.repository -}} {{- if .Values.nodeAnalyzer.runtimeScanner.image.digest -}} @ {{- .Values.nodeAnalyzer.runtimeScanner.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.runtimeScanner.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper Sysdig nodeAnalyzer image name for the Eve Connector
+*/}}
+{{- define "nodeAnalyzer.image.eveConnector" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.runtimeScanner.eveConnector.image.repository -}} {{- if .Values.nodeAnalyzer.runtimeScanner.eveConnector.image.digest -}} @ {{- .Values.nodeAnalyzer.runtimeScanner.eveConnector.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.runtimeScanner.eveConnector.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper image name for the Image Analyzer
+*/}}
+{{- define "nodeAnalyzer.image.imageAnalyzer" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.imageAnalyzer.image.repository -}} {{- if .Values.nodeAnalyzer.imageAnalyzer.image.digest -}} @ {{- .Values.nodeAnalyzer.imageAnalyzer.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.imageAnalyzer.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper image name for the Host Analyzer
+*/}}
+{{- define "nodeAnalyzer.image.hostAnalyzer" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.hostAnalyzer.image.repository -}} {{- if .Values.nodeAnalyzer.hostAnalyzer.image.digest -}} @ {{- .Values.nodeAnalyzer.hostAnalyzer.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.hostAnalyzer.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper image name for the Benchmark Runner
+*/}}
+{{- define "nodeAnalyzer.image.benchmarkRunner" -}}
+    {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.benchmarkRunner.image.repository -}} {{- if .Values.nodeAnalyzer.benchmarkRunner.image.digest -}} @ {{- .Values.nodeAnalyzer.benchmarkRunner.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.benchmarkRunner.image.tag -}} {{- end -}}
+{{- end -}}
+
+{{/*
+Daemonset labels
```

Comment:
Since this is referring to the `daemonset` map, this is probably not used or necessary